src/cli/credentials_create.rs

```diff
@@ -33,6 +34,18 @@ impl Command for crate::cli::CredentialsCreate {
             .category(Category::Custom("couchbase".to_string()))
             .switch("read", "enable read access", None)
             .switch("write", "enable write access", None)
+            .named(
+                "username",
+                SyntaxShape::String,
+                "the username to use for the registered cluster",
+                None,
+            )
+            .named(
+                "password",
+                SyntaxShape::String,
+                "the password to use with the registered cluster",
```

Comment:
I would maintain parallelism between the two instructions so using **for** in both instances

```suggestion
                "the password to use for the registered cluster",
```